drivers/video/ov7670.c

```diff
@@ -220,7 +220,7 @@ static const struct ov7670_reg ov7670_init_regtbl[] = {
 
 	/* configure the output timing */
 	/* PCLK does not toggle during horizontal blank, one PCLK, one pixel */
-	{OV7670_COM10, 0x20}, /* COM10 */
+  {OV7670_COM10, 0x03}, /* COM10 */
```

Comment:
Please avoid unneeded formatting changes. Also please rebase the branch and not merge main into it, to avoid clutter due to merge commits. You don't generally have to update the branch unless asked to do so or if there are merge conflicts

```suggestion
	{OV7670_COM10, 0x03}, /* COM10 */
```